docs/getting_started/authentication.md

```diff
@@ -0,0 +1,40 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Authentication
+
+The Authentication mechanism is based on a bearer token.
+How does it work?
+Both Jellyfish and Business Logic are configured with the same secret.
+This secret is used for authorizing HTTP requests between them (via HTTP authorization request header).
+When a new client wants to join a room, Business Logic (assuming the client is authorized to join) requests adding a new peer and receives a token generated by Jellyfish.
+The token must then be passed to the client, that will use it to connect to the room as a new peer. 
+The token encodes both peer and room IDs.
+The token exchange is handled by our Client SDKs when connecting.
+Under the hood, Jellyfish expects the token in the first message after opening the Web Socket connection.
+
+Token encodes peer id and room id. 
+
+1. Run Jellyfish
+
+```bash
+TOKEN=HTTP_AUTH_TOKEN mix phx.server
+```
+
+2. Create a room and add one peer
+
+```elixir title="Business Logic"
+client = Jellyfish.Client.new("http://address-of-your-server.com", "mySecret")
```

Comment:
Since the http token passed to the Jellyfish via `TOKEN` enviroment variable is the same one that we should authenticate the http requests with, i think they should have the same value in this example.
```suggestion
client = Jellyfish.Client.new("http://address-of-your-server.com", "HTTP_AUTH_TOKEN")
```